src/main/java/io/thorntail/example/GreetingEndpoint.java

```diff
@@ -13,38 +13,34 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package io.openshift.boosters;
+package io.thorntail.example;
 
-import java.util.Optional;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
+import javax.ws.rs.DefaultValue;
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.Response;
-
-import org.eclipse.microprofile.config.inject.ConfigProperty;
+import java.util.Optional;
```

Comment:
But even `java.` is at the bottom. Well, since no specific order is enforced for these boosters, it works either way :-)

Reply:
That's the IntelliJ style, yes :-)